gcapi/gcapi.py

```diff
@@ -578,6 +610,10 @@ def upload_cases(
             The slug of the archive to use.
         reader_study
             The slug of the reader study to use.
+        process
+            Should the images be processed (start a job for the algorithm or add to
+            archive or reader study). Set to False when using an algorithm with
+            multiple inputs.
```

Comment:
I think we can remove this kwarg. Don't give the users the option, we can remove the legacy endpoint. If the have multiple images they want to run then they can loop around this function. Make processing far more efficient.